src/pyseekdb/utils/embedding_functions/google_vertex_embedding_function.py

```
@@ -0,0 +1,130 @@
+    def name() -> str:
+        return "google_vertex"
+
+    @staticmethod
+    def build_from_config(config: dict[str, Any]) -> "GoogleVertexEmbeddingFunction":
+        api_key_env = config.get("api_key_env")
+        model_name = config.get("model_name")
+        project_id = config.get("project_id")
+        region = config.get("region")
+
+        if api_key_env is None or model_name is None or project_id is None or region is None:
+            raise ValueError("Missing required configuration")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)